app/scripts/views/form.js

```diff
@@ -94,7 +94,11 @@ function (_, $, p, Validate, AuthErrors, BaseView, Tooltip,
       return values;
     },
 
-    enableSubmitIfValid: function () {
+    enableSubmitIfValid: function (event) {
+      if (event) {
+        event.stopImmediatePropagation();
```

Comment:
Does it make sense to just add this to `BaseView.cancelEventThen`? Would that break anything?